doc/stages/writers.ept_addon.rst

```diff
@@ -0,0 +1,66 @@
+.. _writers.ept_addon:
+
+writers.ept_addon
+=================
+
+The **EPT addon writer** supports writing additional dimensions to `Entwine Point Tile`_ datasets.  Thus the EPT addon writer is unique among most other PDAL writers due to its limitation that it may only be used along with an `EPT reader`_ and modifies and existing dataset rather than creating a brand new one.
```

Comment:
"...modifies ~and~ an existing dataset..."

But it doesn't really modify anything. The addon dimension stands alone, but is necessarily linked to an existing asset.

Reply:
Language fixed in c57730d.